lib/util/print.js

```diff
@@ -91,4 +96,25 @@ function printTreeTraversal(node) {
   }
 }
 
+// Need to add parens around any [+, -] operation
+// next to [/, *, ^] operation or parenthesis node
```

Comment:
I'd replace 'next to' with 'nested in' or something showing that it's above it in the tree, not beside it